components/SEO.js

```diff
@@ -24,6 +24,11 @@ const CommonSEO = ({ title, description, ogType, ogImage, twImage }) => {
       <meta name="twitter:title" content={title} />
       <meta name="twitter:description" content={description} />
       <meta name="twitter:image" content={twImage} />
+      <link
+        rel="canonical"
+        href={canonicalUrl ? canonicalUrl : `${siteMetadata.siteUrl}${router.asPath}`}
+        key={'canonicalUrl'}
```

Comment:
Why do we need `key` here?

Reply:
Because I didn't want to remove the `canonical` tag at `BlogSEO` but removed later.